File: crates/siloscan-core/tests/profiles-corpus/tree/ruby/positive.rb
def f
  g
rescue Exception => e
  h(e)
end
def f
  x = g rescue nil
  x
end
def f(a)
  a = a
  a
end
def f(c)
  if c
    1
  else
    1
  end
end
def f
  g
ensure
  return 1
end

File: crates/siloscan-core/tests/profiles-corpus/tree/csharp/positive.cs
class C { void M() { try { G(); } catch (System.Exception) {} } }
class C { bool M(int a) { return a == a; } }
class C { void M() { try { G(); } catch (System.Exception e) { throw e; } } }
class C { async void M() { await G(); } }
class C { int M(bool c) { if (c) { return 1; } else { return 1; } } }
